corehq/apps/geospatial/tests/test_es.py

```diff
@@ -25,6 +25,7 @@ def fake_get_max_doc_count(query, case_property, precision):
     return 2 * x + 9_983
 
 
+@patch('corehq.apps.geospatial.es.MAX_GEOHASH_DOC_COUNT', 10000)
```

Comment:
Should this be 10000 or 1000?

Reply:
Any of that should work inpractice. 
I added 10000 as the test case originally was for 10000 and the faker `fake_get_max_doc_count` uses value close to 10000.